jormungandr-lib/src/interfaces/vote/mod.rs

```diff
@@ -0,0 +1,69 @@
+use serde::Serialize;
+
+#[derive(Serialize)]
+#[serde(remote = "chain_impl_mockchain::block::BlockDate")]
+struct BlockDate {
+    pub epoch: u32,
+    pub slot_id: u32,
+}
+
+#[derive(Serialize)]
+#[serde(remote = "chain_impl_mockchain::certificate::ExternalProposalId")]
+struct ExternalProposalId {
+    #[serde(getter = "chain_impl_mockchain::certificate::ExternalProposalId::to_string")]
+    id: String,
+}
+
+#[derive(Serialize)]
+pub struct Proposal {
+    pub external_id: String,
```

Comment:
This is a `Hash` I believe. We already got a type for this

Reply:
But `Hash` doesn't implement `serde::Serialize`, so I wanted to keep it simple instead of making more types.

Reply:
Ok, was looking at other `Hash`. I found the one I should use. Thanks!